inc/Loading/LoadListener.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2018 United States Government as represented by the
+// Administrator of the National Aeronautics and Space Administration.
+// All Rights Reserved.
+#ifndef LoadListener_h
+#define LoadListener_h
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "Loading/LoadEstimator.h"
+#include "Messages/MessageBus.h"
+#include "Messages/VectorMessage.h"
+#include "Models/PrognosticsModel.h"
+
+namespace PCOE {
+    class LoadListener: public IMessageProcessor {
+    public:
+
+        LoadListener(MessageBus& bus, const std::string& src,LoadEstimator* le) : bus(bus), le(le) {
```

Comment:
```suggestion
        LoadListener(MessageBus& bus, const std::string& src, LoadEstimator* le) : bus(bus), le(le) {
```

Reply:
Also, I think `LoadEstimator` can be a reference instead of a pointer.

Reply:
Switched to reference and added space